CMakeLists.txt

```diff
@@ -483,8 +495,11 @@ find_package(Threads REQUIRED)
 find_package(OIS REQUIRED)
 find_package(OGRE REQUIRED)
 find_package(CEGUI REQUIRED)
-find_package(SFML 2 REQUIRED COMPONENTS Audio System Network)
-
+if(UNIX)
+    find_package(SFML 2 REQUIRED COMPONENTS Audio System Network Window Graphics)
```

Comment:
The other dependencies should likely be added only if `OD_USE_SFML_WINDOW` is true.